modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -27,24 +27,23 @@ public static class Common {
         /** Unknown error group. */
         public static final ErrorGroup COMMON_ERR_GROUP = ErrorGroup.newGroup("CMN", 1);
 
-        /** Unexpected error. */
-        public static final int UNEXPECTED_ERR = COMMON_ERR_GROUP.registerErrorCode(1);
-
         /** Node stopping error. */
-        public static final int NODE_STOPPING_ERR = COMMON_ERR_GROUP.registerErrorCode(2);
+        public static final int NODE_STOPPING_ERR = COMMON_ERR_GROUP.registerErrorCode(1);
 
         /** Component not started error. */
-        public static final int COMPONENT_NOT_STARTED_ERR = COMMON_ERR_GROUP.registerErrorCode(3);
+        public static final int COMPONENT_NOT_STARTED_ERR = COMMON_ERR_GROUP.registerErrorCode(2);
 
         /** Illegal argument or argument in a wrong format has been passed. */
-        public static final int ILLEGAL_ARGUMENT_ERR = COMMON_ERR_GROUP.registerErrorCode(4);
+        public static final int ILLEGAL_ARGUMENT_ERR = COMMON_ERR_GROUP.registerErrorCode(3);
 
         /** SSL can not be configured error. */
-        public static final int SSL_CONFIGURATION_ERR = COMMON_ERR_GROUP.registerErrorCode(5);
+        public static final int SSL_CONFIGURATION_ERR = COMMON_ERR_GROUP.registerErrorCode(4);
 
-        /** Unknown error. */
-        @Deprecated
-        public static final int UNKNOWN_ERR = COMMON_ERR_GROUP.registerErrorCode(0xFFFF);
+        /**
+         * This error code represents an internal error caused by faulty logic or coding in the Ignite codebase.
+         * In general, this error code should be considered as a non-recoverable error
+         */
+        public static final int INTERNAL_ERR = COMMON_ERR_GROUP.registerErrorCode(0xFFFF);
```

Comment:
By the way, we already have INTERNAL_ERR in SQL_ERR_GROUP.

Reply:
Changed all occurrences to Common.INTERNAL_ERR. The Sql.INTERNAL_ERR removed.